packages/mco/components/data-policies/create-dr-policy/select-cluster-list.tsx

```diff
@@ -134,18 +165,17 @@ export const SelectClusterList: React.FC<SelectClusterListProps> = ({
           isCompact
           className="mco-select-cluster-list__data-list"
         >
-          {filteredClusters.map((fc) => (
-            <DataListItem key="">
+          {filteredClusters.map((fc, index) => (
+            <DataListItem key="select cluster">
```

Comment:
Keys should be unique for each item. It helps react to learn the order when reconciling.
```suggestion
            <DataListItem key={fc?.name}>
```